pages/privacy-policy/index.html

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Privacy Policy
-modified_date: September 29, 2020
+modified_date: June 03, 2021
```

Comment:
Shouldn't this date change along with the policy itself, and only the new entry to the feed itself happen in this commit? I haven't done this before, though, so we may need confirmation from @josh-padnick.

Reply:
I just looked through the history for this file a bit, which seems to confirm my suspicion. We probably want to lump the date change in with the first commit, and then update the follow-up accordingly with just the feed update (and the new hash).

Reply:
@ebeneliason I don't understand. The hash of the commit shows only the privacy policy updates and not updates to unrelated pages.

Reply:
https://github.com/gruntwork-io/gruntwork-io.github.io/pull/433/commits/acefb523d621da17d2d445221b2f071c1c50efa5

Reply:
Well, there are multiple problems now, but the initial issue I'm pointing out here is that this commit (58adbc98ccdb6b152989509ee1735c33430945fe) is intended to do nothing other than add the entry to the feed, with a hash of the commit that represents the change to the policy. However, you're updating the modification date here as well. The update to the date should happen in the same commit as the policy change itself, and the follow-up commit should only modify the feed.

The issue is now further complicated by the follow-up commits needed for the policy document itself. You'll need to rebase the changes, squash the policy fixes into the initial policy change commit, and update the date as well. Then you can take the hash of that, update the feed entry accordingly, and then again squash that down so you're left with 2 clean commits.

1. all policy changes + date
2. feed update with hash from 1